src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -39,3 +61,20 @@ func start(baseSettings *Settings) error {
 	service, err = ui.Register(messageType, nil, helloFn)
 	return err
 }
+
+func read() {
+	for msg := range service.In {
+		settings := (msg).(map[string]interface{})
+		config.Update(func(updated *config.Config) error {
+
+			autoReport := settings["autoReport"].(bool)
+			if autoReport {
+				analytics.Start()
```

Comment:
It would probably be better for analytics to have the typical Configure() method that gets called on config updates and then start/stop itself based on autoReport.  That way, if someone else changes the setting, it still works.